src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -164,8 +165,25 @@ class CollapsibleCourseRun extends React.Component {
     }
   }
 
-  displayDefaultButtonLabel(status, isAdmin) {
```

Comment:
nit: Leave passing in isAdmin since we already pull out administrator inside of render and it can avoid the three lines here.